fix fictional, correct some content & whitespace

scr0llbaer9000 opened the pull request:
A few more things can be done in this phase.

* Fix wrong values for fictional resp. add the tag in user files where it was missing, as discussed
* Fix long-standing indentation/whitespace issues in `database_people.xml`
* Some minor content corrections
* Add missing IMDB data to the remain real-people entries in [speedminister.xml](https://github.com/TVTower/TVTower/compare/master...scr0llbaer9000:TVTower:res-update-2b-fix-fictional?expand=1#diff-7de15ca888e729f8e17e4435b2e4ff1d7a850b7ee060d9cca8a336fe6b583bf2), to be moved in subsequent step.

No entries have have been moved in this commit.

GWRon commented on res/database/Default/user/kasi.xml:
```diff
@@ -1243,7 +1243,7 @@
 
-		<!--Episoden gekürzt, kein optionaler Kult-->
+		<!--Episoden gekürzt (TODO: Klären Warum?), kein optionaler Kult-->
 		<scripttemplate licence_type="3" product="2" guid="Kasi-script-Metalsucher">
```
On the look for "Metal" (music) or "Metall" (the resource).

Think it got cut down to avoid a 20 episodes series.

nittka replied:
Only 3 episodes were left out. Feel free to add them. A general review for episodes would be nice at some point (more variety in titles and descriptions).

scr0llbaer9000 replied:
> Feel free to add them

ok, but at a later date. Requires a closer look. The episode content seems to overlap.

GWRon commented on res/database/Default/user/speedminister.xml:
```diff
@@ -21,10 +21,10 @@
-        
-        <person id="SpeedMinister-person-id037" first_name="Den" last_name="Jakobson" first_name_original="Dennis" last_name_original="Jacobsen" creator="8605" created_by="speedminister" />
-        <person id="SpeedMinister-person-id038" first_name="Lianda" last_name="Mahoud" first_name_original="Randa" last_name_original="Chahoud" creator="8605" created_by="speedminister" />
+        <!-- TODO: Most non-fictional people in this file seem to be unused? Update: IMDB data added, TODO: move entries, TODO: add related programme, e.g. tt0978537: Ijon Tichy: Raumpilot -->
```
Think SpeedMinister added them first and then created some programme for them.

Normally we do not really need the unused ones - because we have a person generator for a variety of countries. Yet of course the pre-generation of certain/specific persons might be interesting (because of hmm more "realistic _and_ common" names).

nittka replied:
I would prefer removing unused persons (not castable) from the database.

scr0llbaer9000 replied:
hmm if someone makes the effort I'd rather prefer to add a TODO to add the related programme.

And in this case, I have found the missing programme. Speedminister had posted a newer version of his data: https://www.gamezworld.de/phpforum/viewtopic.php?pid=79070#p79070

I'll compare it and add it at a later date.

GWRon commented:
The changes until this post seem to be Okay/mergeable.

nittka commented on res/database/Default/database_programmes.xml:
```diff
@@ -9666,60 +9666,59 @@ Dream scenes and sentiments warp the storyline.</en>
 					<title>
-						<de>Episode 1.2</de>
-						<en>Episode 1.2</en>
+						<de>Der Geheimnisvolle Prinz</de>
```
adjective lower case?

scr0llbaer9000 replied:
[Title casing](https://en.wikipedia.org/wiki/Title_case) is handled very inconsistently in general in the game data, in both German in English. (But it's not much better in the real world either.)

Agree in this case though, the other episodes are also not title-cased in German.

scr0llbaer9000 commented:
In a 3rd commit (not showing in this thread likely because I had not pushed it, committed only locally) I moved the non-fictional people data from speedminister.xml to `database_people.xml`. I made no other changes to any data. https://github.com/TVTower/TVTower/pull/913/commits/5ac6934faefd3bcabb54fa0c116ed9f67d3e0042

In the 4th commit I made the minor correction discussed above, and added some more `fictional` information/tags to a few entries, the case that was discussed here https://github.com/TVTower/TVTower/pull/911#issuecomment-1430248394 (no guarantee I caught all though)

nittka commented:
Do you want to add further commits to this PR?

scr0llbaer9000 commented:
no, next changes should go into new PRs.
